manifest.json

```diff
@@ -58,7 +58,16 @@
               "tags": {
                 "4.7.1-$(RuntimeReleaseDateStamp)-windowsservercore-ltsc2016": {},
                 "4.7.1-windowsservercore-ltsc2016": {}
-              }
+              },
```

Comment:
Friendly reminder to pull in latest and the changes that added 4.8.  You will need to declare dependencies on 4.8 as well right?